.github/workflows/build-wheels.yml

```diff
@@ -151,17 +145,10 @@ jobs:
           python -m pip install --upgrade pip
         CIBW_ARCHS_MACOS: ${{ matrix.os.cibw-archs-macos[matrix.arch.matrix] }}
         CIBW_BEFORE_ALL_MACOS: >
-          brew install boost cmake
+          brew install gmp boost cmake
```

Comment:
don't need gmp but not a big deal. i think my cp from main obliterated this fix

Reply:
removed